gbus/tx/mysql/txoutbox.go

```diff
@@ -332,7 +343,8 @@ func (outbox *TxOutbox) ensureSchema(tx *sql.Tx, svcName string) error {
 	delivery_tag	bigint(20) NOT NULL,
 	delivery_attempts int NOT NULL DEFAULT 0,
 	insert_date	timestamp DEFAULT CURRENT_TIMESTAMP,
-	PRIMARY KEY(rec_id))`
+	PRIMARY KEY(rec_id),
+	INDEX status_delivery (rec_id, status, delivery_attempts))`
```

Comment:
If you change the order of the index to 
```suggestion
	INDEX status_delivery (status, delivery_attempts, rec_id))`
```
you would not need to git the `USE INDEX (status_delivery)` hint in the query.